hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/functional/TestOrcBootstrap.java

```diff
@@ -246,6 +247,8 @@ private void testBootstrapCommon(boolean partitioned, boolean deltaCommit, Effec
             .withBootstrapParallelism(3)
             .withBootstrapModeSelector(bootstrapModeSelectorClass)
             .withBootstrapModeForRegexMatch(modeForRegexMatch).build())
+        .withMetadataConfig(HoodieMetadataConfig.newBuilder().enable(true).withMaxNumDeltaCommitsBeforeCompaction(3)
```

Comment:
[HUDI-8774](https://issues.apache.org/jira/browse/HUDI-8774) track bootstrap support for col stats